### PR TITLE
Add references for n-dimensional arrays

### PR DESCRIPTION
**Pull request by Crissman**
Improve documentation by changing `numpy.ndarray or cupy.ndarray` references to link to `n-dimensional arrays` page.

### CODE REVIEW

**Comment by Crissman** [comment]
Extension of #6042

File: chainer/function.py [hunk 1]
```diff
@@ -249,11 +249,9 @@ def __call__(self, *inputs):
         behavior of building the computational graph.
 
         Args:
-            inputs: Tuple of input :class:`Variable`, :class:`numpy.ndarray` or
-                :class:`cupy.ndarray` objects.
-                If the input is an :class:`numpy.ndarray` or a
-                :class:`cupy.ndarray`, it is automatically wrapped with
-                :class:`Variable`.
+            inputs: Tuple of input :class:`Variable`, :ref:`ndarray` objects.
```
Comment (toslunar) [inline chainer/function.py]: ',' -> 'or'?

**Comment by Crissman** [comment]
Fixed. Thanks!

**Comment by toslunar** [comment]
Jenkins, test this please.

**Pull request merged**